src/test/java/uk/gov/hmcts/reform/civil/service/LocationRefDataServiceTest.java

```diff
@@ -174,6 +178,29 @@ void shouldReturnLocations_whenLRDReturnsAllLocations() {
                 .isEqualTo("service_token");
     }
 
+    @Test
```

Comment:
don't think this is needed either as we have it in master already?

Reply:
Again, I don't know why it still appears as new code since it is your code